docs/topics_5/web-adding-global-custom-rules.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='adding_global_custom_rules_{context}']
+= Adding global custom rules
+
+Global rules apply to all projects and are automatically applied to analyses for which they fit the definitions of a project.
```

Comment:
Break this sentence into two sentences.

Not sure what 'for which they fit the definitions of a project" means. It's very murky but we don't have time to fix this, so just break up the sentences.

Reply:
Chopped off the murky part. "GRs apply to all projects." Same with global labels.